.github/workflows/ci.yml

```diff
@@ -141,3 +141,66 @@ jobs:
     - name: mypy (typecheck)
       run:  mypy -p mpi4py
               --config-file=conf/mypy.ini
+
+  skbuild:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        py:
+          - 3
+        mpi:
+          - mpich
+          - openmpi
+        os:
+          - ubuntu-18.04
+          - ubuntu-20.04
+
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup MPI (${{ matrix.mpi }})
+      uses: mpi4py/setup-mpi@v1
+      with:
+        mpi: ${{ matrix.mpi }}
+
+    - name: Use Python ${{ matrix.py }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.py }}
+
+    - name: Install packaging tools
+      run:  python -m pip install build
+
+    - name: Build sdist
+      run:  python -m build -s
+      env:
+        MPI_BUILD_BACKEND: skbuild
```

Comment:
Shouldn't this be `MPI4PY_BUILD_BACKEND`?

Reply:
Yes, it should, great catch!. Although to build an sdist, it does not really matter. I'll fix it in an upcoming PR.